### DISCUSSION TITLE
Uninstallation of exe takes too long

### DISCUSSION THREAD

**Discussion by Diwakar243** [wixtoolset/issues#6823] [Questions]
* Which version of WiX are you building with?

> WiX 3.11.2

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> WiX Toolset Visual Studio Extension version 3.11.2

* Which version of .NET are you building with?

> 4.6.1

* Describe the problem and the steps to reproduce it.

> Our application which is created using WiX toolset takes too long to install/uninstall. Can I get a solution to improve the installation and uninstallation performance.

* Describe the behavior you expected and how it differed from the actual behavior.

> Expecting some improvement in the installation and uninstallation performance.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by rseanhall** [comment]
Please provide more information. For example, the log that it creates in %TMP% would be a good start.
